_source/_posts/2021-10-xx-accidental-dev-to-advocate.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: blog_post
+title: "Spreading some Okta love to the DevOps world"
+author: deepu-sasidharan
+by: advocate
+communities: [java, javascript, go, rust, devops, announcement]
+description: "Welcome Deepu K Sasidharan to the Okta team!"
+tags: [kubernetes, devops, java, go, rust, javascript, typescript, okta]
+tweets:
+  - "Welcome @deepu105 to the @oktadev team!"
+  - "Excited to have @deepu105 join the @oktadev team!"
+  - "🎉 @deepu105 joins the @oktadev team to spread some Okta love to the DevOps world"
+type: awareness
+---
+
+Hello Oktaverse!
+
+So finally, I have landed at Okta, in my second attempt 😉. I'm so excited about this new chapter in my career journey and can't wait to see what's in store. But first introductions.
+
+{% img blog/hello-deepu/hello.jpg alt:"My face" width:"800" %}{: .center-image }
+
+## Who is this guy?
+
+I'm from the south of India, a village in Kerala to be specific, but I grew up in Chennai since my parents moved there looking for work when I was 12. I like to call myself a software engineer with a soldering iron. That's because I never intended to be in the IT industry. I wanted to be an astronomer, but that didn't work out, so I wanted to get into robotics. To pursue that, I got myself an engineering degree in electronics and electrical engineering, but apparently, the universe had different plans. The recession of 2008 hit and Robotics companies in India weren't hiring much. So, to make ends meet and to repay my student loan, I applied to one of the biggest software companies in India with little to no hope. The only software experience I had at that point was the data-structures class which I flunked out and had to retake and a little bit of dabbling with JS and PHP when I ran a personal blog about modifying motorbikes. Surprisingly, I made the cut and hence began my journey into software engineering with a boot camp provided by the company, Tata Consultancy Services (TCS). That is when I realized I was quite good at programming and I passed out of the boot camp as the top performer. I still love learning about astronomy and astrophysics among other science subjects and still ride a motorcycle 😄. I also love Linux and you can read all about that [here](https://deepu.tech/the-state-of-linux-on-desktops/) if you are interested.
```

Comment:
```suggestion
I'm from the south of India, a village in Kerala to be specific, but I grew up in Chennai since my parents moved there looking for work when I was 12. I like to call myself a software engineer with a soldering iron. That's because I never intended to be in the IT industry. I wanted to be an astronomer, but that didn't work out, so I wanted to get into robotics. To pursue that, I got myself an engineering degree in electronics and electrical engineering, but apparently, the universe had different plans. The recession of 2008 hit and Robotics companies in India weren't hiring much. So, to make ends meet and to repay my student loan, I applied to one of the biggest software companies in India with little to no hope. The only software experience I had at that point was the data-structures class which I flunked out and had to retake and a little bit of dabbling with JS and PHP when I ran a personal blog about modifying motorbikes. Surprisingly, I made the cut and hence began my journey into software engineering with a boot camp provided by the company, Tata Consultancy Services (TCS). That is when I realized I was quite good at programming and I passed out of the boot camp as the top performer. I still love learning about astronomy and astrophysics among other science subjects and still ride a motorcycle 😄 . I also love Linux and you can read all about that [here](https://deepu.tech/the-state-of-linux-on-desktops/) if you are interested.
```
```suggestion
I'm from the south of India, a village in Kerala to be specific, but I grew up in Chennai since my parents moved there looking for work when I was 12. I like to call myself a software engineer with a soldering iron. That's because I never intended to be in the IT industry. I wanted to be an astronomer, but that didn't work out, so I wanted to get into robotics. To pursue that, I got myself an engineering degree in electronics and electrical engineering, but apparently, the universe had different plans. 

The recession of 2008 hit and robotics companies in India weren't hiring much. To make ends meet and to repay my student loan, I applied to one of the biggest software companies in India with little to no hope. The only software experience I had at that point was the data-structures class (which I flunked and had to retake) and a little bit of dabbling with JS and PHP, when I ran a personal blog about modifying motorbikes. Surprisingly, I made the cut. Hence began my journey into software engineering with a boot camp provided by the company, Tata Consultancy Services (TCS). That is when I realized I was quite good at programming and I passed out of the boot camp as the top performer! I still love learning about astronomy and astrophysics, among other science subjects, and still ride a motorcycle 😄. I also love Linux and you can read [my take on Linux](https://deepu.tech/the-state-of-linux-on-desktops/) if you are interested.
```
small edits to increase readability and flow. Some shorter paragraphs and sentences